Makefile

```diff
@@ -48,19 +46,10 @@ SHIM_GO_LDFLAGS=-ldflags '-s -w -X $(PKG)/version.Version=$(VERSION) -X $(PKG)/v
 
 TESTFLAGS_RACE=
 GO_GCFLAGS=
-GO_BUILD_FLAGS=
```

Comment:
Could leave this if people want to manually add `-i`

Reply:
@mlaventure Great idea! I myself will probably be using that as it made a big build time difference.

Reply:
@mlaventure I left both, the (empty) setting and the using of the variable, but I'll remove the first, as it'll be easy to add the options from the command line than editing the Makefile, is that ok?

Reply:
It all depends if you want `GO_BUILD_FLAGS=-i make` to work. If you don't remove it, only `make GO_BUILD_FLAGS=-i` will have the expected effect